zerver/webhooks/front/tests.py

```diff
@@ -0,0 +1,282 @@
+from typing import Text
+import ujson
+
+from zerver.lib.test_classes import WebhookTestCase
+
+class FrontHookTests(WebhookTestCase):
+    STREAM_NAME = 'front'
+    URL_TEMPLATE = "/api/v1/external/front?&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'front'
+
+    def _test_no_message_data(self, fixture_name: Text) -> None:
+        payload = self.get_body(fixture_name)
+        payload_json = ujson.loads(payload)
```

Comment:
Hmm.. why do we need to call `ujson.loads` here? I'm not sure if it should be necessary here.